resources/js/Components/City.vue

```diff
@@ -0,0 +1,229 @@
+<script setup>
+import { onMounted, reactive, ref } from 'vue'
+import { router, useForm } from '@inertiajs/vue3'
+
+const props = defineProps({
+  city: Object,
+  providers: Object,
+})
+
+function destroyCity(cityId) {
+  router.delete(`/cities/${cityId}`)
+}
```

Comment:
Are we displaying info about delete? Any confirmation?

Reply:
It will be covered by frontend dev.